braintree.go

```diff
@@ -5,17 +5,26 @@ import (
 	"encoding/xml"
 	"errors"
 	"fmt"
+	"io"
 	"log"
 	"net/http"
+	"net/url"
 )
 
-const LibraryVersion = "0.10.0"
-
 type apiVersion int
 
+type HTTPClient interface {
+	Do(req *http.Request) (res *http.Response, err error)
+	Get(url string) (res *http.Response, err error)
+	Post(urstring, bodyType string, body io.Reader) (res *http.Response, err error)
+	PostForm(url string, values url.Values) (res *http.Response, err error)
+	Head(url string) (res *http.Response, err error)
+}
+
 const (
-	apiVersion3 apiVersion = 3
-	apiVersion4            = 4
+	apiVersion3    apiVersion = 3
+	apiVersion4               = 4
+	LibraryVersion            = "0.10.0"
```

Comment:
The change moving the `LibraryVersion` looks unnecessary and groups unrelated identifiers together. Could we remove that from the PR diff?